earth_enterprise/src/fusion/portableglobe/cutter/cgi-bin/core/glc_assembler.py

```diff
@@ -601,8 +602,14 @@ def AssembleGlc(self, form_):
       utils.PrintAndLog("SUCCESS", logger, None)
 
       utils.PrintAndLog("Create polygon file: %s" % self.polygon_file, logger)
-      utils.CreateFile(self.polygon_file, spec["polygon"])
-      utils.PrintAndLog("SUCCESS", logger, None)
+
+      with open(self.polygon_file, "w") as fp:
```

Comment:
This looks good, though I was wondering if it could be encapsulated into a function, like the code was before.

Reply:
The with statement adds more complexity to a previous clean interface. Let's add a new function "WritePolygonFile" and hide the file management in there to preserve the clarity inside this function.